src/client/templates/export.html

```diff
@@ -0,0 +1,15 @@
+<template name="exportRoom">
```

Comment:
Was thinking more along the lines of
```
<template name="exportRoom">
    <body>
        <textarea rows="50" cols="200" style="margin: 10px 10px 10px 10px" readonly>
            {{#each getCategories}}
                {{this}}
                {{#each getCards this}}
                        Thought: {{text}}
                        Tags: {{tags}}
                        Author: {{author}}
                        Comments: {{comments}}
                        Likes: {{likes}}

                {{/each}}
            {{/each}}
        </textarea>
    </body>
</template>
```
Edit probably to make it look nicer. Wouldn't need to do any styling changes then.

Would probably add a "Copy to clipboard" button
Check this guy out
http://stackoverflow.com/a/30810322